esrally/driver/runner.py

```diff
@@ -2592,9 +2592,16 @@ async def run_stream(self, es, stream, connection_limit):
                     async with connection_limit:
                         async with runner:
                             response = await runner({"default": es}, item)
-                            timing = response.get("dependent_timing") if response else None
-                            if timing:
-                                timings.append(timing)
+                            if response:
+                                # TODO: support calculating dependent's throughput
```

Comment:
Sorry, not sure I fully grok. We already drop the `unit` and `weight` for dependent tasks in `master`, this comment was more so for discoverability and in case someone else ends up investigating the same code path.

We retain the ops/weight for the actual composite task itself here:
https://github.com/elastic/rally/blob/469df4286d900203843956548274ead46141cc68/esrally/driver/runner.py#L2622-L2631